common/v2/features/Dashboard/Dashboard.tsx

```diff
@@ -79,6 +80,7 @@ export default function Dashboard() {
                 className="Dashboard-desktop-modifiedPanel"
                 copyable={true}
                 dashboard={true}
+                privacyTriggerEnabled={false}
```

Comment:
Is this necessary with your default value being `false`?

Reply:
nah